src/main/java/spireMapOverhaul/SpireAnniversary6Mod.java

```diff
@@ -509,7 +509,7 @@ public void receivePostRender(SpriteBatch sb) {
     private static final float DROPDOWN_X = 400f;
     private static final float DROPDOWN_Y = 600f;
     private ModLabeledToggleButton filterCheckbox;
-    private static final float CHECKBOX_X = 650f;
+    private static final float CHECKBOX_X = 800f;
```

Comment:
I made this change because the checkbox was hidden behind the dropdown for me. This is probably missing scaling or something like that.

It will probably need a better fix eventually but I didn't want to take the time to investigate right now.